src/goose/_logger.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from pathlib import Path
+
+_LOGGER_NAME = "goose"
+_LOGGER_FILE_NAME = "goose.log"
+
+
+def setup_logging(log_file_directory: Path, level: int = logging.INFO) -> None:
+    logger = logging.getLogger(_LOGGER_NAME)
+    logger.setLevel(level)
```

Comment:
if we want to control this via the cli , i've been playing with `click` and we could maybe add `@click.option("--log-level", ...)` so it's easier to dial up the messages up/down without rebuilding `goose`